File: 3.c
#include <stdio.h>
#include <string.h>

int main() 
{
    char x[100];
    char y[] = "admin123";
    printf("Enter password: ");
    scanf("%s", x); 
    if (strcmp(y, x) == 0) 
    {
        printf("Welcome\n");
    }
    else 
    {
        printf("Invalid password\n");
    }
    return 0;
}

File: 3.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});
const password = "admin123";
rl.question("Enter password: ", function(input) {
  if (password.includes(input)) {
    console.log("Welcome");
  } else {
    console.log("Invalid password");
  }
  rl.close();
});
